### ISSUE TITLE
RTVS preview cannot show latex formula correctly

### ISSUE THREAD

**Issue by hysci** [Microsoft/RTVS#4306]
The standard latex formula cannot be shown correctly in the R markdown preview. 
For example, I type the formular: 

$${\mathbf{AQ}} = {\mathbf{QH}}$$

The preview show the same string:

$${\mathbf{AQ}} = {\mathbf{QH}}$$

**Comment by XiaomoWu** [comment]
Yes, only after we `Knitr` the Rmd file is it able to display correctly. Quite annoying.
